### ISSUE TITLE
hasTapGestureRecognizer should check if isEnabled == true

### ISSUE THREAD

**Issue by davideme** [kif-framework/KIF#1118]
When calling `viewContainingAccessibilityElement` with `mustBeTappable` at true, I expect that the `hasTapGestureRecognizer` check if it has on TapGestureRecognizer and is enable.

**Comment by justinseanmartin** [comment]
Is there a specific scenario that you have that isn't working? Can you describe it? 

`hasTapGestureRecognizer` looks like it should be call if `viewContainingAccessibilityElement:tappable:error:` is called with `mustBeTappable` set to `YES`.
